web/yo/app/scripts/services/additionalFile.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Created by jiaojiao on 10/24/17.
+ */
+'use strict';
+
+/**
+ * @ngdoc service
+ * @name oncokb.additionalFile
+ * @description
+ * # Additional Files, currently including meta and queues.
+ * Service in the oncokb.
+ */
+angular.module('oncokbApp')
+    .service('additionalFile', function additionalFile($rootScope, $q, storage, mainUtils, documents, dialogs) {
+        function load(types) {
```

Comment:
Seems like everytime the load function is triggered, we retrieve the file from Google. Should we just keep the meta, and curationQueues alive throughout the application? Basically check whether $rootScope.metaRealtime  and $rootScope.queuesRealtime are available first?

Reply:
good point! We should check if it's available first.